wideq/client.py

```diff
@@ -84,10 +88,11 @@ def __init__(self,
         self._gateway: Optional[core.Gateway] = gateway
         self._auth: Optional[core.Auth] = auth
         self._session: Optional[core.Session] = session
+        self._last_device_update = datetime.now()
 
         # The last list of devices we got from the server. This is the
         # raw JSON list data describing the devices.
-        self._devices: List[Dict[str, Any]] = []
+        self._devices = None
```

Comment:
This change leads to the whole script crashing in case there is no device. Since we later assume _devices to be an interable.
```
  File "./example.py", line 33, in ls
    for device in client.devices:
  File "/home/frederik/dev/projects/home_automation/wideq/wideq/client.py", line 151, in devices
    return (DeviceInfo(d) for d in self._devices)
TypeError: 'NoneType' object is not iterable
```